addons/portal/views/portal_templates.xml

```diff
@@ -282,7 +282,7 @@
 
     <template id="portal_record_sidebar" name="My Portal Record Sidebar">
         <div t-attf-class="#{classes}">
-            <div class="o_portal_sidebar_content sticky-lg-top d-lg-inline-block mb-4 mb-lg-0 p-3 p-lg-0" id="sidebar_content">
+            <div class="o_portal_sidebar_content sticky-lg-top d-lg-inline-block mb-4 mb-lg-0 mt-lg-n5 p-3 p-lg-0 pt-lg-5" id="sidebar_content">
```

Comment:
While it's a nice improvement, price is now touching the header, and needs more spacing. 
Maybe it needs some custo here in my opinion.